src/components/Survey/Survey.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {useEffect, useContext, useState} from 'react';
```

Comment:
I need to understand how we'll map the questions onto different pages. We can discuss in standup. This looks great!